cypress/integration/hw_lesson_15/second_test.js

```diff
@@ -0,0 +1,97 @@
+import { secondTestChangeUser, secondTestUser, currentUser } from '../../second_test/second_test_data.js';
+import { checkValue, changeValue, searchBox, sortUsers } from '../../second_test/second_test_method.js';
+
+describe('HW Lesson 15', () => {
+    it('Cypress navigation', () => {
+        cy.visit('https://demoqa.com/webtables')
+
+        // Should be on a new URL which includes '/webtables'
+        cy.url().should('include', '/webtables')
+
+        cy.get('.btn-primary').click()
+
+        secondTestUser.forEach(item => checkValue(item["selector"], item["value"]));
+
+        cy.get('#submit').click()
+
+        cy.get('.rt-tr-group')
+            .should('contain', secondTestUser[0].value)
+            .and('contain', secondTestUser[1].value)
+            .and('contain', secondTestUser[2].value)
+            .and('contain', secondTestUser[3].value)
+            .and('contain', secondTestUser[4].value)
+            .and('contain', secondTestUser[5].value)
+            .contains(secondTestUser[0].value).parent().find('.mr-2').click();
+
+    })
+
+    it('Edit user and check that each field is editable and delete', () => {
+
+        secondTestChangeUser.forEach(item => changeValue(item["selector"], item["value"]));
+
+        cy.get('#submit').click()
+
+        cy.get('.rt-tr-group')
+            .should('contain', secondTestChangeUser[0].value)
+            .and('contain', secondTestChangeUser[1].value)
+            .and('contain', secondTestChangeUser[2].value)
+            .and('contain', secondTestChangeUser[3].value)
+            .and('contain', secondTestChangeUser[4].value)
+            .and('contain', secondTestChangeUser[5].value)
+            .contains(secondTestChangeUser[0].value).parent().find('.mr-2').next().click();
+    })
+
+    it('check that user was deleted', () => {
+
+        cy.contains(
+            secondTestChangeUser[0].value,
+            secondTestChangeUser[1].value,
+            secondTestChangeUser[2].value,
+            secondTestChangeUser[3].value,
+            secondTestChangeUser[4].value,
+            secondTestChangeUser[5].value
+        ).should('not.exist')
+
+        
+
+    })
+
+    it('Check searching feature, check that appropriate user can be searched by each field', () => {
+
+        searchBox(currentUser['currentUserFirstName']);
+        searchBox(currentUser['currentUserLastName']);
+        searchBox(currentUser['currentUserEmail']);
+        searchBox(currentUser['currentUserAge']);
+        searchBox(currentUser['currentUserSalary']);
+        searchBox(currentUser['curentUserDepartment']);
+    })
+
+    it('should verify table data is sorted ascending', () => {
```

Comment:
This task not full. U need to add verification for all columns

Reply:
do you mean sort check?